app/jobs/upvs/fetch_services_with_forms_list_job.rb

```diff
@@ -36,11 +36,9 @@ def self.source
   def each_row_as_attributes(csv_file, csv_options)
     CSV.foreach(csv_file, csv_options) do |row|
       row = row.to_h.transform_keys { |k| k.to_s.gsub(/\p{Cf}|"/, '') }
-
-      row[row.keys.first].sub!(/\A"/, '')
-      row[row.keys.last].sub!(/"\z/, '')
-
```

Comment:
Ok, funguje to aj takto, ale preferujem pouzit presne rovnake spracovanie ako v tomto [jobe](https://github.com/slovensko-digital/harvester.ecosystem/blob/master/app/jobs/upvs/fetch_public_authority_edesks_list_job.rb) (funguje to s nim), nech to mame jednotne (potom to vieme jednoducho aj v oboch pripradoch upravit, ked bude treba).

Reply:
toto upravim